hw/ip/lc_ctrl/data/lc_ctrl.hjson

```diff
@@ -5,57 +5,459 @@
   clock_primary: "clk_i",
   bus_device: "tlul",
   bus_host: "none",
+  regwidth: "32",
 
-//  ///////////////////////////
-//  // Interrupts and Alerts //
-//  ///////////////////////////
-//
-//  interrupt_list: [
-//  ],
-//
-//  alert_list: [
-//  ],
-//
-//  ////////////////
-//  // Parameters //
-//  ////////////////
-//  param_list: [
-//  ]
-//
-//  /////////////////////////////
-//  // Intermodule Connections //
-//  /////////////////////////////
-//  // TODO: these need to be refined during implementation and integration
-//  inter_signal_list: [
-//  ] // inter_signal_list
-
-  ///////////////
-  // Registers //
-  ///////////////
 
-  regwidth: "32",
+  ///////////////////////////
+  // Interrupts and Alerts //
+  ///////////////////////////
+
+  // This module has no interrupts
+  no_auto_intr_regs: "True"
+
+  alert_list: [
+    { name: "lc_programming_failure",
+      desc: "This alert triggers if an error occurred during an OTP programming operation.",
+    }
+    { name: "lc_state_failure",
+      desc: "This alert triggers if an error in the life cycle state or life cycle controller FSM is detected.",
+    }
+  ],
+
+  ////////////////
+  // Parameters //
+  ////////////////
+
+  param_list: [
+    { name:    "NumTokenWords",
+      desc:    "Number of 32bit words in a token.",
+      type:    "int"
+      default: "4",
+      local:   "true"
+    }
+    { name:    "NumLcStateBits",
+      desc:    "Number of life cycle state enum bits.",
+      type:    "int"
+      default: "4",
+      local:   "true"
+    }
+    { name:    "NumLcCntBits",
+      desc:    "Number of life cycle state enum bits.",
+      type:    "int"
+      default: "5",
+      local:   "true"
+    }
+  ]
+
+  /////////////////////////////
+  // Intermodule Connections //
+  /////////////////////////////
+
+  inter_signal_list: [
+    // life cycle state broadcast from OTP
+    { struct:  "otp_lc_data"
+      type:    "uni"
+      name:    "otp_lc_data"
+      act:     "rsp"
```

Comment:
isn't this supposed to be 'rcv' if it's uni and input only?

Reply:
oh good point, these have to change.